python/cuspatial/cuspatial/core/spatial/bounding.py

```diff
@@ -51,9 +51,7 @@ def polygon_bounding_boxes(polygons: GeoSeries):
     # by combining the geometry offset and parts offset of the multipolygon
     # array.
 
-    poly_offsets = polygons.polygons.part_offset.take(
-        polygons.polygons.geometry_offset
-    )
+    poly_offsets = polygons.polygons.part_offset
```

Comment:
This change specifically breaks multi-polygon point in polygon testing, because it discards the notion we talked about yesterday that quadtree pip has no notion of multipolygons, and we take advantage of that at the python layer.

I think this effectively collapses MultiPolygons into Polygons, combining all the rings of the MultiPolygon into a single polygon. I'm not sure that's correct behavior anyway because then the exterior ring of the second polygon in a MultiPolygon becomes an interior ring, right?

Reply:
Thanks, this is clear now. Can you add a line in the doc to mention that multipolygon bounding box is unsupported now?